keyboards/snampad/config.h

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
I should have actually told you three pull requests before, but I'll allow this because I gave you bad/incomplete information. Generally our standard is that keyboard submissions should be isolated to their own pull requests, unless the addition is multiple keyboards designed by the same person.

What I should have told you before was:

- one pull request to fix the JC65 if Backlight and RGB are both disabled
- a second pull request to add the Snampad keyboard
- a third pull request to add all your personal keymaps

I apologize for this confusion.